src/pages/Payment.vue

```diff
@@ -2,8 +2,87 @@
+                    </div>
+                  </label>
+                </div>
+
+                <!-- Tab row -->
+                <div class="tab row">
+                  <input id="tab-two" type="checkbox" name="tabs">
+                  <label for="tab-two">
+                    <div class="row">
+                      <div>1/2/17</div><div>36</div><div class="amount">£36</div>
+                    </div>
+                  </label>
+                  <div class="tab-content">
+                    <div class="tab">
+                      <label for="">
+                        <div class="row-code">
+                          <div class="code">RVP12345561</div><div class="date">12/1/17</div>
+                        </div>
+                        <div class="row-code">
+                          <div class="code">RVP12345563</div><div class="date">12/1/17</div>
+                        </div>
+                      </label>
+                    </div>
+                  </div>
+                </div>
+
+                <!-- Tab row -->
+                <div class="tab row">
+                  <input id="tab-three" type="checkbox" name="tabs">
+                  <label for="tab-three">
+                    <div class="row">
+                      <div>13/3/17</div><div>68</div><div class="amount">£68</div>
+                    </div>
+                  </label>
+                  <div class="tab-content">
+                    <div class="tab">
+                      <label for="">
+                        <div class="row-code">
+                          <div class="code">RVP12345561</div><div class="date">12/1/17</div>
+                        </div>
+                        <div class="row-code">
+                          <div class="code">RVP12345563</div><div class="date">12/1/17</div>
+                        </div>
+                      </label>
+                    </div>
+                  </div>
+                </div>
+
+                <!-- Tab row -->
+                <div class="tab row">
+                  <input id="tab-four" type="checkbox" name="tabs">
+                  <label for="tab-four">
+                    <div class="row">
+                      <div>28/4/17</div><div>41</div><div class="amount">£41</div>
+                    </div>
+                  </label>
+                  <div class="tab-content">
+                    <div class="tab">
+                      <label for="">
+                        <div class="row-code">
+                          <div class="code">RVP12345561</div><div class="date">12/1/17</div>
+                        </div>
+                        <div class="row-code">
+                          <div class="code">RVP12345563</div><div class="date">12/1/17</div>
+                        </div>
+                      </label>
+                    </div>
+                  </div>
+                </div>
+
+              </div>
```

Comment:
Yup, they'll all be vue'd with for-in statements. JSON is cooking. then SQL/Eloquent to make it happen.